web/js/roomView.js

```diff
@@ -384,14 +385,16 @@
       }
     });
 
-    var dialOutBtn = document.getElementById('dialOutBtn');
-    dialOutBtn.addEventListener('click', function(event) {
-      event.preventDefault();
-      Utils.sendEvent('roomView:dialOut', {
-        phoneNumber: document.getElementById('dialOutNumber').value
+    if (enableSip) {
```

Comment:
I added this because otherwise there it didn't find the `dialOutBtn` when sip is disabled and threw an error breaking the whole app.